### PR TITLE
[meebitsdao-delegation] New MeebitsDAO voting strategy

### PR DESCRIPTION
**Pull request by lightninglu10**
Changes proposed in this pull request:
- New strategy to use for DAO voting for https://vote.meebitsdao.com/#/
- Uses the formula: 

MAX(1, # of Meebits (limit 20) * Delegate Tokens)

### CODE REVIEW

**Comment by lightninglu10** [comment]
hey @ChaituVR i committed your suggestion

File: src/strategies/meebitsdao-delegation/index.ts [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+  );
+
+  const meebitsScores = await erc721Strategy(
+    space,
+    options.tokenAddresses[2].network,
+    getProvider(options.tokenAddresses[2].network),
+    addresses,
+    options.tokenAddresses[2],
+    blocks[options.tokenAddresses[2].network]
+  );
```
[the remark this answers is not in the document]
Comment (maikir) [reply src/strategies/meebitsdao-delegation/index.ts:72]: Hi @ChaituVR. So I'm not sure I'm understanding things 100%, but you want us to have a strategy param, and call meegitsdao-delegation 3 times (one with our first strategy as the strategy param, one with our second strategy as the strategy param, and one with our third strategy as the strategy param). That sounds fine, but how would we add up the scores or manipulate them afterwards together if the meebitsdao-delegation is called 3 times separately? Can we cache or store the results somehow for the next strategy to use?

[the remark this answers is not in the document]
Comment (maikir) [reply src/strategies/meebitsdao-delegation/index.ts:72]: The issue is we are calling three separate strategies, but we are kind of combining the results (not just adding them up).

[the remark this answers is not in the document]
Comment (maikir) [reply src/strategies/meebitsdao-delegation/index.ts:72]: oh do we use the getScoresDirect thing?

**Comment by maikir** [comment]
@ChaituVR I've implemented getScoresDirect(). Unfortunately its a bit messy because we use different addresses for the three strategies (two strategies use the same set, one strategy uses one set). Hope this is okay? For getScoresDirect() we can implement multiple strategies in an array, but can only input one set of addresses (same for all strategies) as its currently implemented. Please advise on any additional changes that might be necessary.

**Comment by maikir** [comment]
hey @ChaituVR, I have reverted the previous commits. Should be back to what we originally were trying to perform!

**Review by ChaituVR — APPROVED** [review]

**Pull request merged**